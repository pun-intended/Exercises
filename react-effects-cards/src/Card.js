
function Card({img}) {
    return(
        <div className="Card">
            <img src={img}></img>
        </div>
    )
}

export default Card