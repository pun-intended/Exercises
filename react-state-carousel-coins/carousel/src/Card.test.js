import React from "react";
import { render, fireEvent } from "@testing-library/react";
import Card from "./Card";

test("renders without crashing", () => {
    render(<Card caption="test" src="./image1.jpg" currNum="3" totalNum="10"/>)
})

it("matches snapshot", () => {
    const { asFragment } = render(
                    <Card caption="test" 
                    src="./image1.jpg" 
                    currNum="3" 
                    totalNum="10"/> )
    expect(asFragment()).toMatchSnapshot()
})