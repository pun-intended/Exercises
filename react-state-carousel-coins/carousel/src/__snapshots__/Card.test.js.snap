// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      test
    </h4>
    <img
      alt="test"
      class="Card-image"
      src="./image1.jpg"
    />
    <small
      class="Card-small"
    >
      Image 3 of 10.
    </small>
  </div>
</DocumentFragment>
`;
